src/SFA.DAS.EAS.Web/Controllers/EmployerCommitmentsController.cs

```diff
@@ -5,6 +5,7 @@
 using System.Web.Mvc;
 using FluentValidation;
 using FluentValidation.Mvc;
+using SFA.DAS.CommitmentsV2.Types;
```

Comment:
I don't think our v1 should reference any v2 types - we could just add the new enum to our v1 for now?

Reply:
It already references it in several places. I don't think it's wrong to reference V2 types library from V1. As it allows us to migrate code